### PR TITLE
unreal_asset: update level export

### PR DESCRIPTION
**Pull request by bananaturtlesandwich**
updates the representation of `LevelExport` to include fields like model, level script actor explicitly. The reason that precomputed visibility handler and precomputed volume distance field aren't also represented is the same rationale behind not including StaticMesh and Texture exports. The extra data is simply too large and in most cases is not needed. If someone is really desparate they can parse the extras like I do [here](https://github.com/bananaturtlesandwich/stove/blob/main/src/extras/mesh.rs) for static meshes

### CODE REVIEW

File: unreal_asset/src/exports/level_export.rs [hunk 1]
```diff
@@ -80,21 +98,32 @@ impl ExportTrait for LevelExport {
 
         asset.write_i32::<LE>(0)?;
         asset.write_i32::<LE>(self.actors.len() as i32)?;
-        for actor in &self.actors {
+        for actor in self.actors.iter() {
```
Comment (localcc) [inline unreal_asset/src/exports/level_export.rs:101]: why the change to iter?

Comment (bananaturtlesandwich) [reply unreal_asset/src/exports/level_export.rs:101]: idk I personally think it's nicer - it's the same thing under the hood anyway but I can change it back if you want

Comment (localcc) [reply unreal_asset/src/exports/level_export.rs:101]: most of the code uses `&` so it's nicer if you would change it back for consistency

Comment (bananaturtlesandwich) [reply unreal_asset/src/exports/level_export.rs:101]: sure
